.github/workflows/client-test.yml

```
@@ -312,7 +312,15 @@ jobs:
       - name: Wait for 30 seconds for server to start
         run: |
             sleep 30s 
-            ps -ef | grep server-1.0-SNAPSHOT
+            
+      - name: Exit if Server hasnt started
+        run: |
+         if [[ `ps -ef | grep "server-1.0-SNAPSHOTwrong" | grep java |wc -l` == 0 ]]; then
```

Just noticed this typo. Please correct this. 

```suggestion
         if [[ `ps -ef | grep "server-1.0-SNAPSHOT" | grep java |wc -l` == 0 ]]; then
```